Ginger/GingerCoreNETUnitTest/RunTestslib/CLITest.cs

```diff
@@ -654,10 +715,13 @@ public void OLDCLIDynamicRegressionTest()
 
                 // Act            
                 CLIProcessor CLI = new CLIProcessor();
-                CLI.ExecuteArgs(new string[] { "Dynamic=" + configFile });
-
-                // Assert            
-                Assert.AreEqual(eRunStatus.Passed, WorkSpace.Instance.RunsetExecutor.Runners[0].BusinessFlows[0].RunStatus, "BF RunStatus=Passed");
+                
+            Task.Run(async () =>
+            {
+                await CLI.ExecuteArgs(new string[] { "Dynamic=" + configFile });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5632404)